tests/statespace_sse_test.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 Google LLC. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "statespace_test.h"
+
+#include "gtest/gtest.h"
+
+#include "../lib/parfor.h"
+#include "../lib/simulator_sse.h"
+#include "../lib/statespace_sse.h"
+
+namespace qsim {
+
```

Comment:
Might want to put these tests in an anonymous namespace or maybe in `simulator_test.h` ?

Reply:
What is the reason to use an anonymous namespace here?

Reply:
It’s usually not a bad idea to put tests and other helpers for tests in anonymous namespace so you don’t accidentally get name collisions on functions or classes etc.